File: TP/TP1/Code/5J-2.cpp
/* Afficher un motif donné */
#include <iostream>

int main()
{
  int j;
  
  for (int i = 0; i < 7; ++i) {
    if (i < 4) {
      for (j = 0; j < 3 - i; ++j)
        std::cout << " " << std::flush;
    } else {
      for (j = 0; j < i - 3; ++j)
        std::cout << " " << std::flush;
    }
    for (int k = 0; k < 7 - j * 2; ++k)
      std::cout << "*" << std::flush;
    std::cout << std::endl;
  }
  
  return 0;
}

File: TP/TP1/Code/Python/5J-2.py
#!/usr/bin/python
# Afficher un motif donné
for i in range(7):
    if i < 4:
        for j in range(4 - i):
            print(' ', end='')
    else:
        for j in range(i - 2):
            print(' ', end='')
    for k in range(7 - j * 2):
        print('*', end='')
    print()  # Ici on affiche un saut de ligne
